README.md

```diff
@@ -20,4 +20,14 @@ This collection requires some additional dependencies to be installed:
 ```shell
 pip install rhoas-sdks --force-reinstall
 ```
-All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found here https://console.redhat.com/openshift/token
\ No newline at end of file
+
+## Using collection
+
+All modules require an 'OFFLINE_TOKEN' environment variable to be set with a valid token. This token is used to authenticate the user. The token is an OpenShift Cluster Manager API Token and can be found [here](https://console.redhat.com/openshift/token)
```

Comment:
This is not secure also will not work on external nodes. Can we have this as argument?

Reply:
As in prompt the user to specify the `OFFLINE_TOKEN` when running the playbook?

Reply:
Token needs to be argument.